programs/stake/src/stake_state.rs

```diff
@@ -64,489 +57,90 @@ pub(crate) fn null_tracer() -> Option<impl FnMut(&InflationPointCalculationEvent
     None::<fn(&_)>
 }
 
-impl Default for StakeState {
-    fn default() -> Self {
-        StakeState::Uninitialized
-    }
-}
-
-impl StakeState {
-    pub fn get_rent_exempt_reserve(rent: &Rent) -> u64 {
-        rent.minimum_balance(std::mem::size_of::<StakeState>())
-    }
-
+pub struct StakeConverter;
```

Comment:
I guess same question here. Could these just be vanilla functions, instead of creating a vacant struct to force them to be associated?

Reply:
They definitely could be -- if the preference is for that, I can do it.  It also differentiates the uses of `solana_stake_program::stake_state` and `solana_sdk::stake::state`.

Reply:
That is my preference, but open to persuasion if others feel differently.

Reply:
No need! It looks much nicer as top-level functions.  I just had to rename a couple of them

Reply:
Done in 0d298a9